.ncurc.yml

```diff
@@ -0,0 +1,8 @@
+reject:
+  # ESM only packages
+  - delay
+  - got
+
+  # Other
+  - koa-hbs
+  - ldapjs
```

Comment:
Please add comments to each of them explaining why they are rejected so we don't need to check over and over.